### ISSUE TITLE
Examples loading should hide after loading is complete.

### ISSUE THREAD

**Issue by Maksims** [playcanvas/engine#5781]
Recently engine examples were reworked which is great, but it affected few examples.
One of the effect is that loading spinner does not hide after loading is complete. This leads to examples with transparent canvas to appear as they infinitely loading.

### Solution:

After example is finished loading it should hide the spinner.

**Comment by mvaligursky** [comment]
@kungfooman - any idea on this one?

**Comment by kungfooman** [comment]
Thank you! Seems to be a relict of not using any actual component state, I will make a PR right away.

Just for testing on my side, which example was affected and iPhone or Android?

**Comment by Maksims** [comment]
> Thank you! Seems to be a relict of not using any actual component state, I will make a PR right away.
> 
> Just for testing on my side, which example was affected and iPhone or Android?

AR Basic.

**Comment by kungfooman** [comment]
@Maksims Can you make a quick check if it works for you?

Old one: https://playcanvas.github.io/#/xr/ar-basic
New one: https://examples.killtube.org/#/xr/ar-basic

**Comment by Maksims** [comment]
Apparently issue only happens when developing locally. Both deployed versions works well.

**Comment by kungfooman** [comment]
> Apparently issue only happens when developing locally. Both deployed versions works well.

That's good, I don't think it's only local, the old one I linked is still the pre-ES6-PR-version (which probably didn't have this issue), so when both work now, then the PR is doing its job.

But if it's really a local-only-thing, you would figure that out by testing the PR - or once the PR is merged. Maybe it even depends on the picked graphics device (for example WebGL1 vs. WebGPU).

Thank you for the feedback! I have a hard time testing the AR examples (it works with Immersive Web Emulator, but I never had this transparent canvas Spinner issue).

**Comment by Maksims** [comment]
> > Apparently issue only happens when developing locally. Both deployed versions works well.
> 
> That's good, I don't think it's only local, the old one I linked is still the pre-ES6-PR-version (which probably didn't have this issue), so when both work now, then the PR is doing its job.
> 
> But if it's really a local-only-thing, you would figure that out by testing the PR - or once the PR is merged. Maybe it even depends on the picked graphics device (for example WebGL1 vs. WebGPU).
> 
> Thank you for the feedback! I have a hard time testing the AR examples (it works with Immersive Web Emulator, but I never had this transparent canvas Spinner issue).

Yeah, testing WebXR stuff requires HTTPS. I had to put nginx and with SSL, to serve it in local network instead of `npm run serve` for testing.
